js/apps/system/_admin/aardvark/APP/frontend/js/views/collectionsView.js

```diff
@@ -592,25 +592,35 @@
             );
           
             if (window.App.isCluster) {
+              var minReplicationFactor = (this.minReplicationFactor ? this.minReplicationFactor : 1); 
+              var maxReplicationFactor = (this.maxReplicationFactor ? this.maxReplicationFactor : 10); 
+
+              // clamp replicationFactor between min & max allowed values
+              var replicationFactor = '';
+              if (properties.replicationFactor) {
+                replicationFactor = parseInt(properties.replicationFactor);
+                if (replicationFactor < minReplicationFactor) {
+                  replicationFactor = minReplicationFactor;
+                }
+                if (replicationFactor > maxReplicationFactor) {
+                  replicationFactor = maxReplicationFactor;
+                }
+              }
+
               tableContent.push(
                 window.modalView.createTextEntry(
                   'new-replication-factor',
                   'Replication factor',
-                  properties.replicationFactor ? properties.replicationFactor : '',
-                  'Numeric value. Must be between ' + 
-                  (this.minReplicationFactor ? this.minReplicationFactor : 1) + 
-                  ' and ' + 
-                  (this.maxReplicationFactor ? this.maxReplicationFactor : 10) +
-                  '. Total number of copies of the data in the cluster',
+                  String(replicationFactor),
```

Comment:
Just unsure, not an objections...
What is the addition of String(replicationFactor)?